dist/tools/openvisualizer/makefile.openvisualizer.inc.mk

```diff
@@ -0,0 +1,68 @@
+.PHONY: openv-clean openv-setroot openv-term openv-termroot
+
+# Use a single board with openv
+# ===================================
+#
+# OpenVisualizer runs on port 9000 by default, if that ports conflicts or
+# multiple instances are spawned you will need to specify the port, eg.
+#   * `OPENV_FLAGS='--port=9001`
+#
+# Not all logs for openvisualizer are piped to the terminal, more detailed logs
+# are stored in $(BINDIR)/openv-server.log
+#
+# Supported:
+#  * openv-term
+#  * openv-termroot
+#  * openv-setroot
+#  * openv-clean
+#
+# Prerequisites
+# -------------
+#
+# * Install openvisualizer:
+#   * git clone -b pr_develop_SW-318_baud_mask_opt https://github.com/fjmolinas/openvisualizer.git
```

Comment:
`develop_SW-318-RIOT`, `pr_develop_SW-318_iotlab`?

Reply:
Ajj I referenced the wrong branch here.

Reply:
I mean the one to checkout is the correct one

Reply:
fixed